src/System.ComponentModel.TypeConverter/tests/TypeConverterTestBase.cs

```diff
@@ -153,6 +166,20 @@ public void GetStandardValuesExclusive_Invoke_ReturnsExpected()
             Assert.Equal(StandardValuesExclusive, converter.GetStandardValuesExclusive());
         }
 
+        private static void AssertEqual(object expected, object actual)
```

Comment:
We should name this differently; otherwise it looks like a typo at the call site, e.g. AssertInstanceDescriptorsEqual, or something like that.